src/reactIntl.js

```diff
@@ -0,0 +1,10 @@
+export const intl = {
+	formatMessage: (trn) => trn.defaultMessage || 'formatted message',
+	formatDate: (date) => date || 'formatted date',
```

Comment:
better to call `date.toString()` to ensure that it is returned with the correct type - same for the other string formatters